packages/init/src/index.ts

```diff
@@ -15,11 +16,20 @@ const CONFIG_PREFIX = '--force';
  */
 
 export default function initializeInquirer(...args: string[]): Function | void {
-    const packages = args;
-    const includesDefaultPrefix = packages.includes(AUTO_PREFIX);
-    const generateConfig = packages.includes(CONFIG_PREFIX);
-    if (packages.length === 0 || includesDefaultPrefix || generateConfig) {
+    const cli = new WebpackCLI();
+    const { opts, unknownArgs } = cli.argParser(options, args, true);
```

Comment:
Why do not fix it all problems here? It is very bad to handle this in the each package?

Reply:
Yeah, makes sense.